admin-api/central_ledger_api_v1.0.json

```diff
@@ -2050,71 +1804,105 @@
-                        "example": "NET"
+                        "$ref": "#/components/schemas/SettlementGranularity"
                     },
                     "settlementInterchange": {
-                        "type": "string",
-                        "enum": [
-                        	"BILATERAL",
-                        	"MULTILATERAL"
-                        ],
-                        "description": "Specifies the type of settlement arrangement between parties. Allowed values for the enumeration are: BILATERAL - Each participant settles its obligations with each other separately, and the scheme is not a party to the settlement. MULTILATERAL - Each participant settles with the scheme for the net of its obligations, rather than settling separately with each other party.",
-                        "example": "MULTILATERAL"
+                        "$ref": "#/components/schemas/SettlementInterchange"
                     },
                     "settlementDelay": {
-                        "type": "string",
-                        "enum": [
-                        	"IMMEDIATE",
-                        	"DEFERRED"
-                        ],
-                        "description": "Specifies if settlement happens immediately after a transfer has completed or with a delay. Allowed values for the enumeration are: IMMEDIATE - Settlement happens immediately after a transfer has completed. DEFERRED - Settlement is managed by the Hub operator on-demand or via a schedule.",
-                        "example": "DEFERRED"
+                        "$ref": "#/components/schemas/SettlementDelay"
                     },
                     "currency": {
                         "$ref": "#/components/schemas/Currency"
                     },
                     "requireLiquidityCheck": {
-                        "type": "boolean",
-                        "description": "Liquidity check boolean"
+                        "$ref": "#/components/schemas/RequireLiquidityCheck"
                     },
                     "ledgerAccountType": {
                         "type": "string",
-                        "description": "Account type for the settlement model POSITION, SETTLEMENT or INTERCHANGE_FEE",
+                        "description": "Account type for the settlement model: POSITION, SETTLEMENT, or INTERCHANGE_FEE.",
                         "enum": [
                             "INTERCHANGE_FEE",
-                            "POSITION"
+                            "POSITION",
+                            "SETTLEMENT"
                         ]
                     },
+                    "settlementAccountType": {
+                        "type": "string",
+                        "description": "A special type of ledger account into which settlements should be settled: SETTLEMENT or INTERCHANGE_FEE_SETTLEMENT.",
+                        "enum": [

```

Comment:
1. I think we should remove the enum restriction on this as it's possible to add additional settlementAccountTypes via the admin API, and those accounts can be handled by a custom Settlement Rule.

Rather change `enum` to `examples` as per the following example:

```
  "examples": [
      "INTERCHANGE_FEE",
      "SETTLEMENT"
  ]
```

2. Remove`POSITION` from the description & example as it is not a valid settlementAccountType. For settling the position, the settlementAccountType `SETTLEMENT` will be used.